main.tf

```
@@ -23,6 +23,7 @@ resource "aws_cloudwatch_log_group" "app" {
 module "alb_ingress" {
   source  = "cloudposse/alb-ingress/aws"
   version = "0.23.0"
+  enabled = var.alb_enabled
```

I think it should be this.

```suggestion
  enabled = module.this.enabled && var.alb_enabled
```

---

It would be safer to have a `local` at the top that has the following

```hcl
locals {
  alb_enabled = module.this.enabled && var.alb_enabled
}
```

and then reference `local.alb_enabled` here and below